### PR TITLE
Make first week.

### PR DESCRIPTION
**Pull request by aces47**
Add vUserTask view.
Add stored procedures SetUserTaskAsFail, SetUserTaskAsSuccess.
Add SeedData for TaskState.

### CODE REVIEW

File: DIMS-Core.Database/DIMS-Core.Database/Tables/TaskState.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[TaskState](
+	[StateId] [int] IDENTITY(1,1) NOT NULL,
+	[StateName] [nvarchar](50) NOT NULL,
+ CONSTRAINT [PK_TaskState] PRIMARY KEY CLUSTERED 
+(
+	[StateId] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Tables/TaskState.sql:8]: Whole this description is redundant and looks like autogenerated. Remove it. Write only PK description.

File: DIMS-Core.Database/DIMS-Core.Database/Tables/UserTask.sql [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+ALTER TABLE [dbo].[UserTask] CHECK CONSTRAINT [FK_UserTask_TaskTrack]
+GO
+
+ALTER TABLE [dbo].[UserTask]  WITH CHECK ADD  CONSTRAINT [FK_UserTask_UserProfiles] FOREIGN KEY([UserId])
+REFERENCES [dbo].[UserProfiles] ([UserId])
+GO
+
+ALTER TABLE [dbo].[UserTask] CHECK CONSTRAINT [FK_UserTask_UserProfiles]
+GO
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Tables/UserTask.sql:39]: Read your task in document. Description of foreign keys must be after column description Why do you need all this ALTER TABLE scripts?

File: DIMS-Core.Database/DIMS-Core.Database/Tables/TaskState.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[TaskState](
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Tables/TaskState.sql:1]: Plural name

File: DIMS-Core.Database/DIMS-Core.Database/Views/vUserTask.sql [hunk 4]
```diff
@@ -0,0 +1,5 @@
+﻿CREATE VIEW [dbo].[vUserTask]
+	AS SELECT [dbo].[UserTask].[UserId], [dbo].[UserTask].[TaskId], Task.Name AS TaskName, Task.Description AS TaskDescription, Task.StartDate, Task.DeadlineDate, [dbo].[TaskState].[StateName]
+FROM            [dbo].[Task] INNER JOIN
+                         [dbo].[UserTask] ON Task.TaskId = [dbo].[UserTask].[UserTaskId] INNER JOIN
+                         [dbo].[TaskState] ON [dbo].[UserTask].[StateId] = [dbo].[TaskState].[StateId]
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Views/vUserTask.sql:5]: Format this pls. All columns have to be wrapped on different rows.

**Review by PushMeNow — CHANGES_REQUESTED** [review]

File: DIMS-Core.Database/DIMS-Core.Database/Tables/TaskState.sql (hunk 1 above)
Comment (aces47) [reply DIMS-Core.Database/DIMS-Core.Database/Tables/TaskState.sql]: Fixed

File: DIMS-Core.Database/DIMS-Core.Database/Tables/UserTask.sql (hunk 2 above)
Comment (aces47) [reply DIMS-Core.Database/DIMS-Core.Database/Tables/UserTask.sql]: Fixed

File: DIMS-Core.Database/DIMS-Core.Database/Tables/TaskTrack.sql [hunk 5]
```diff
@@ -0,0 +1,6 @@
+﻿CREATE TABLE [dbo].[TaskTracks](
+	[TaskTrackId] [int] PRIMARY KEY IDENTITY(1,1) NOT NULL,
```
Comment (kuzmiich) [inline DIMS-Core.Database/DIMS-Core.Database/Tables/TaskTrack.sql:2]: You can add PK to the end of the file. Tell me, if we leave it as

**Review by PushMeNow — APPROVED** [review]

**Pull request merged**